### PR TITLE
Installation instructions for IBM Z - format fixes

### PR DESCRIPTION
**Pull request by zzoubkova**
Installation instructions for IBM Z - format fixes according to the comments from the original PR and from IBM team.

### CODE REVIEW

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -93,22 +97,25 @@ Each cluster machine must meet the following minimum requirements:
 |120 GB
 
 |Compute
-|{op-system}
+ifdef::ibm-z[|{op-system}]
+ifndef::ibm-z[|{op-system} or RHEL 7.6]
```
Comment (jaypoulz) [inline modules/installation-requirements-user-infra.adoc]: We don't support rhel 7.6 workers for s390x ... yet :)

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+coreos.inst.ignition_url=ftp://cl1.provide.example.com:8080/ignition-bootstrap-0
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1
+rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0 cio_ignore=all,
+!condev rd.dasd=0.0.3490
+----
+
+. Transfer the initramfs, kernel, parameter files, and {op-system} images to z/VM, for example with FTP. For details about how to transfer the files with FTP and boot from the virtual reader, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-installing-zvm-s390[Installing under Z/VM].
+. Punch the files to the virtual reader of the z/VM guest virtual machine that is to become your bootstrap node.
++
+For details about the PUNCH command, see https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.dmsb4/pun.htm
```
Comment (jaypoulz) [inline modules/installation-ibm-z-user-infra-machines-iso.adoc]: Might be good to format this line the same we format the IPL link below.

File: modules/installation-requirements-user-infra.adoc (hunk 1 above)
Comment (jaypoulz) [reply modules/installation-requirements-user-infra.adoc]: This is correct - as we talked about in slack My bad.

**Review by jaypoulz — APPROVED** [review]
With the worker node fix and the formatting of the punch command, I feel confident that this changeset is good to go. :)

So I'll leave this preemptively.

LGTM

File: modules/installation-ibm-z-user-infra-machines-iso.adoc (hunk 2 above)
Comment (zzoubkova) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc]: Fixed in dba335b7d70b2f89ea3d8cfb77cbf10607f7de47 Thank you! :)

File: modules/installation-approve-csrs.adoc [hunk 3]
```diff
@@ -27,6 +31,19 @@ these CSRs are approved or, if necessary, approve them yourself.
 
 . Confirm that the cluster recognizes the machines:
 +
+ifdef::ibm-z[]
+----
+# oc get nodes
+
+NAME 			        STATUS  ROLES    AGE   VERSION
+master-0.cl1mstr0.example.com   Ready   master   20h   v1.14.6+888f9c630
```
Comment (kalexand-rh) [inline modules/installation-approve-csrs.adoc]: We'll need a follow-up for master/4.3 to update the version number.

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+.Prerequisites
+
+* An FTP server running on your provisioning machine that is accessible to the machines you create.
+
+.Procedure
+
+. Log in to Linux on your provisioning machine.
+
+. Download the Red Hat Enterprise Linux CoreOS installation files from the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.2/latest/[{op-system} image mirror].
```
Comment (kalexand-rh) [inline modules/installation-ibm-z-user-infra-machines-iso.adoc]: This link will be different for 4.3, but I'll open a PR to fix it at GA.

File: modules/installation-requirements-user-infra.adoc [hunk 5]
```diff
@@ -93,22 +97,25 @@ Each cluster machine must meet the following minimum requirements:
...
 |120 GB
 
 |===
 
-ifeval::["{context}" == "installing-ibm-z"]
+ifdef::ibm-z[]
 [id="minimum-ibm-z-system-requirements_{context}"]
 == Minimum IBM Z system requirements
 
+You can install OpenShift 4.2 on IBM Z hardware as of z13, z13s, or LinuxONE.
```
Comment (kalexand-rh) [inline modules/installation-requirements-user-infra.adoc]: s/OpenShift 4.2/{product-title} version {product-version}

"IBM Z hardware as of z13, z13s, or LinuxONE" doesn't scan right to me. Does "on the following IBM Z hardware types: z13, z13s, or LinuxONE" sound right to you?

**Review by kalexand-rh — COMMENTED** [review]
This looks great! I have one small change, then it should be good after we squash.

File: modules/installation-ibm-z-user-infra-machines-iso.adoc (hunk 4 above)
Comment (zzoubkova) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc]: Thank you!

File: modules/installation-requirements-user-infra.adoc (hunk 5 above)
Comment (zzoubkova) [reply modules/installation-requirements-user-infra.adoc]: I'd change it to: "You can install OpenShift 4.2 on IBM LinuxONE or one of the following IBM Z hardware products: z13, z13s." 

Wdyt @kalexand-rh @dbenoit17?

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: I think that looks good!

Comment (kalexand-rh) [reply modules/installation-requirements-user-infra.adoc]: @dbenoit17, does this look ok to you?

> You can install OpenShift 4.2 on IBM LinuxONE or one of the following IBM Z hardware products: z13, z13s.

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+that match your {product-title} version if they are available.
+====
++
+Download the following files:
+
+* The initramfs: `rhcos-<version>-installer-initramfs.img`
+* The kernel: `rhcos-<version>-installer-kernel`
+* The operating system image for the disk on which you want to install {op-system}. This type can differ by virtual machine:
++
+`rhcos-<version>-dasd.raw.gz` for DASD
```
Comment (kalexand-rh) [inline modules/installation-ibm-z-user-infra-machines-iso.adoc]: @zzoubkova, per https://github.com/jhradilek/openshift-docs/commit/53595164cede4ba7699bc02ee397567c6cd74bac#r36452811, this should be  `rhcos-<version>-s390x-metal-dasd.raw.gz`

Comment (zzoubkova) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc]: Fixed in 36c8aa640b1c0301f102b11713b57ea33b720242

File: modules/installation-requirements-user-infra.adoc (hunk 5 above)
Comment (zzoubkova) [reply modules/installation-requirements-user-infra.adoc]: Rephrased in 36c8aa640b1c0301f102b11713b57ea33b720242 
Maria wanted to add the HW as it was missing and this was her suggestion. I would add more HW later if needed, but for now we have the info there, which is imporant.

Comment (dbenoit17) [reply modules/installation-requirements-user-infra.adoc]: We also support the z14 and z15 platforms.  Could we add those to the list?

Thanks!

**Review by dbenoit17 — CHANGES_REQUESTED** [review]

File: modules/installation-requirements-user-infra.adoc [hunk 7]
```diff
@@ -109,8 +109,7 @@ ifdef::ibm-z[]
 [id="minimum-ibm-z-system-requirements_{context}"]
 == Minimum IBM Z system requirements
 
-You can install {product-title} version {product-version} on IBM LinuxONE 
-or one of the following IBM Z hardware products: z13, z13s.
+You can install {product-title} version {product-version} on following versions of IBM Z or IBM LinuxONE hardware: z13 or higher, LinuxONE z13 or higher.
```
Comment (kalexand-rh) [inline modules/installation-requirements-user-infra.adoc]: Maybe 

> If you use IBM Z hardware, you can install {product-title} version {product-version} if you use z13 or later. If you use IBM LinuxONE hardware, you can install it on LinuxONE z13 or later.

**Pull request merged**